docs/source/tutorials/base/how_to_train/classification.rst

```diff
@@ -1,7 +1,234 @@
 
-<Guidelines for writing>
+This live example shows how to easily train, validate, optimize and export classification model on the `flowers dataset <https://www.tensorflow.org/hub/tutorials/image_feature_vector#the_flowers_dataset>`_ from TensorFlow.
 
+.. note::
 
-This tutorial explores the OTX classification task that could be used for standard, hierarchical, and multi-label use cases.
\ No newline at end of file
+  To learn deeper how to manage training process of the model, refer to :doc:`./detection`.
+
+  To learn how to deploy the trained model, refer to: :doc:`../deploy`.
+
+  To learn how to run the demo and visualize results, refer to: :doc:`../demo`.
+
+The process has been tested on the following configuration.
+
+- Ubuntu 20.04
+- NVIDIA GeForce RTX 3090
+- Intel(R) Core(TM) i9-10980XE
+- CUDA Toolkit 11.1
+
+.. note::
+
+  While this example shows how to work with :doc:`multi-class classification <../../../explanation/algorithms/classification/multi_class_classification>`, it is easy to extend it for the :doc:`multi-label <../../../explanation/algorithms/classification/multi_label_classification>` or :doc:`hierarchical <../../../explanation/algorithms/classification/multi_label_classification>` classification.
+  Substitute the dataset with a multi-label or hierarchical one. Everything else remains the same.
+
+
+*************************
+Setup virtual environment
+*************************
+
+You can follow the installation process from a :doc:`quick start guide <../../../get_started/installation>` to create a universal virtual environment for OTX.
+
+***************************
+Dataset preparation
+***************************
+
+1.  Download and prepare a `flowers dataset <https://www.tensorflow.org/hub/tutorials/image_feature_vector#the_flowers_dataset>`_ with the following command:
+
+.. code-block::
+
+  cd data
+  wget http://download.tensorflow.org/example_images/flower_photos.tgz
+  tar -xzvf flower_photos.tgz
+  cd ..
+
+
+.. image:: ../../../../utils/images/flowers.jpg
+  :width: 600
+
+
+This dataset contains images of 5 different flower categories and is stored in the imagenet format which is supported by OTX:
+
+.. code-block::
+
+  flower_photos
+    ├── daisy
+    ├── dandelion
+    ├── roses
+    ├── sunflowers
+    ├── tulips
+
+
+*********
+Training
+*********
+
+1. First of all, we need to choose which classification model will we train.
+The list of supported templates for classification is available with the command line below.
+
+.. note::
+
+  The characteristics and detailed comparison of the models could be found in :doc:`Explanation section <../../../explanation/algorithms/classification/multi_class_classification>`.
+
+  We also can modify the architecture of supported models with various backbones, please refer to the :doc:`advanced tutorial for model customization <../../advanced/backbones>`.
+
+.. code-block::
+
+  (otx) ...$ otx find --template --task CLASSIFICATION
```

Comment:
Added a PR https://github.com/openvinotoolkit/training_extensions/pull/1668 that can also work with the `otx find --task classification` command.